persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -182,36 +190,35 @@ public void execute() {
                     ScriptRunner sr = new ScriptRunner(connection);
                     sr.runQuery(query);
                 } catch (SQLException e) {
-                    errStream.println("ERROR: failed to create the database(" +
-                            persistConfigurations.getDbConfig().getDatabase() + "). " + e.getMessage());
+                    errStream.printf("ERROR: failed to create the database(%s). %s%n",
+                            persistConfigurations.getDbConfig().getDatabase(), e.getMessage());
                     return;
                 }
-                errStream.println("Created database `" + persistConfigurations.getDbConfig().getDatabase() + "`.");
+                errStream.printf("Created database '%s'.", persistConfigurations.getDbConfig().getDatabase());
 
-                String sqlFilePath = Paths.get(this.sourcePath, PERSIST_DIRECTORY,
-                                String.format(PersistToolsConstants.SQL_SCHEMA_FILE, entityModule.getModuleName()))
-                        .toAbsolutePath().toString();
+                String sqlFilePath = generatedSourceDirPath.resolve(String.format(PersistToolsConstants.SQL_SCHEMA_FILE,
+                                entityModule.getModuleName())).toAbsolutePath().toString();
                 try (Connection connection = getDBConnection(driver, persistConfigurations, true);
                      Reader fileReader = new BufferedReader(new FileReader(sqlFilePath,
                              StandardCharsets.UTF_8))) {
                     ScriptRunner sr = new ScriptRunner(connection);
                     sr.runScript(fileReader);
                 } catch (IOException e) {
-                    errStream.println("error occurred while reading SQL schema file, "
-                            + sqlFilePath + ". " + e.getMessage());
+                    errStream.printf("ERROR: failed to read SQL schema file(%s). %s%n. ",
+                            sqlFilePath, e.getMessage());
                     return;
                 } catch (Exception e) {
-                    errStream.println("ERROR: failed to read the SQL schema file("
-                            + sqlFilePath + "). " + e.getMessage());
+                    errStream.printf("ERROR: failed to read the SQL schema file(%s). %s%n.", sqlFilePath,
+                            e.getMessage());
                     return;
                 }
-                errStream.println("Created tables for definition in " + file.getFileName() + " in the database `" +
-                        persistConfigurations.getDbConfig().getDatabase() + "`.");
+                errStream.printf("Created tables for definition in %s in the database '%s'. ", file.getFileName(),
+                        persistConfigurations.getDbConfig().getDatabase());
```

Comment:
new line character is missing

Reply:
updated in [4a4bd31](https://github.com/ballerina-platform/persist-tools/pull/140/commits/4a4bd31fb65a4ef430fc372c49473ee14bd4eaac)